src/wordpress/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:buster
+
+RUN apt-get update && \
+  apt-get --yes --no-install-recommends install php7.3 php-mysqli php-fpm wget sendmail \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /var/lib/wp
+
+EXPOSE 9000
+
+COPY ./conf/www.conf /etc/php/7.3/fpm/pool.d
+COPY ./bin /usr/local/bin
```

Comment:
I'm confused with this line 😅

Reply:
by which line are you confused?

Reply:
`COPY ./bin /usr/local/bin`

I guess here you are making sure the binaries are the same in the container but don't we want to use the ones installed in the container

Reply:
Oh, this copies the content of the `./src/wordpress/bin` folder to the `/usr/local/bin` of the Docker image.
`/usr/local/bin` may not have a trailing `/`, otherwise it copies everything at `/usr/local/bin/bin`.
However, I do not think it overwrites the content of `/usr/local/bin`. Anyhow, most of the time `/usr/local/bin` is empty. Linux distributions should not install anything there. It is supposed to be used for custom installations like here.